README.md

```diff
@@ -89,6 +89,28 @@ const app = new App({
 })
 ```
 
+> **NOTE**: an optional `expiry` parameter is passed to `cache.set`, useful for further automation of your cache mechanism
+> _e.g. auto purge values after expiry is elapsed._
+
+```js
+const app = new App({
+  id: APP_ID,
+  privateKey: PRIVATE_KEY,
+  cache: {
+    get (key) {
+      return CACHE[key]
+    },
+    set (key, value, expiry) {
+      CACHE[key] = value
+      
+      const timeout = (new Date(expiry)).getTime() - (new Date()).getTime();
+
+      setTimeout(() => delete CACHE[key], timeout)
+    }
+  }
+})
+```
```

Comment:
maybe instead of adding the additional example, add the `expiry` option to the first example and mention what it does in the description, and that it’s a timeout in ms?

Reply:
And maybe call it `maxAge` to align with the naming of https://github.com/isaacs/node-lru-cache?

Reply:
going the `maxAge` route, means manipulating the original value as passed by GitHub, which I have two thoughts about:

1. if converting to `maxAge`, we know (today) that's 1hr, so it becomes redundant to convert the `expires_at` value, might as well just pass `3.6e+6`
2. in case of network delays, db write delay, etc ... a `1 hr` value is not accurate, might change by a second or two (or more! :scream:)

maybe we keep `maxAge` as is, and pass a 4th optional argument? other option is to pass the `response.data` object as is and allow for the use to decide what to do with it. _(which means modifying the default LRU lib calls accordingly)_